module/data/actor/character.mjs

```diff
@@ -118,6 +118,9 @@ export default class CharacterData extends CreatureTemplate {
           }),
           failure: new NumberField({
             required: true, nullable: false, integer: true, min: 0, initial: 0, label: "DND5E.DeathSaveFailures"
+          }),
+          bonuses: new SchemaField({
```

Comment:
Do we foresee other bonuses being added? If not we can probably just go with `bonus` (similar to `attributes.init.bonus`).

Also make sure to add this to the JSDoc above the class.

Reply:
I was of the opinion that `init` was the outlier and that the general schema should be `bonuses` with `.check`, `.save` or both. The same is done in `attributes.concentration`.